consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/blockcreation/BftMiningCoordinator.java

```diff
@@ -119,6 +152,41 @@ public void stop() {
     }
   }
 
+  @Override
+  public void subscribe() {
+    if (syncState == null) {
+      return;
+    }
+    syncState.subscribeSyncStatus(
+        syncStatus -> {
+          if (syncState.syncTarget().isPresent()) {
+            // We're syncing so stop doing other stuff
+            LOG.info("Stopping QBFT mining coordinator while we are syncing");
```

Comment:
Should remove QBFT from these messages, as this might not always be QBFT as this class is also used in ibft2